src/command/expire/expire.c

```diff
@@ -900,68 +899,103 @@ cmdExpire(void)
                 THROW_FMT(OptionInvalidValueError, "'%s' is not a valid backup label format", strZ(adhocBackupLabel));
         }
 
+        // Track any errors that may occur
+        unsigned int errorTotal = 0;
```

Comment:
Seems like a simple boolean would do the job.

Reply:
Kept as count and added it to final THROW. Commit https://github.com/pgbackrest/pgbackrest/pull/1308/commits/54e3bf08da92f1c157dc6d3f79f80439e4f35265